### PR TITLE
Improve page load speed 

### PR DESCRIPTION
**Pull request by devodii**
### What did i do? 
I've used the **Suspense** and **Lazy** Loading component from react to dynamically import the loading component so as to improve the load speed of the homepage 

### Why? 
* For a better user experience 
* Faster load times can result in lower bounce rates and often leads to an increase in conversion rate

#### Tough, this was at a very low level. But I assume the point is clear

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -22,3 +22,9 @@ dist-ssr
 *.njsproj
 *.sln
 *.sw?
+
```
Comment (Njong392) [inline .gitignore:25]: 🤔 I see there is now a `.env` in the `.gitignore` file. Did you by any chance create secrets in the course of your changes?

**Review by Njong392 — CHANGES_REQUESTED** [review]
Hey there! Thank you so much for the new changes to Abbreve 🥇 

1. About the lazy loading, I think we created a custom loading component to handle that. It handles the loading state of the button. Is it not double work to wrap the same button with `Suspense` again? Maybe I am the one missing something. Please enlighten me haha
2. I see you have made some unwarranted changes to the `package-lock.json` file. Please revert this to avoid any conflicts in the future.
3. Every other change you made looks great! Thank you for the sweet, sweet scroll bar 🤩

File: .gitignore (hunk 1 above)
Comment (devodii) [reply .gitignore:25]: Oh... This, supposed to be for another project. 
My bad :(

**Comment by devodii** [comment]
Very correct, All fixed!

**Comment by Njong392** [comment]
Ehm... I see no changes, sorry. What did you fix?

**Pull request closed without merging**